### PR TITLE
3992 new module openmspeakpickerhires

### LINKED ISSUE #3992: new module: openms/peakpickerhires
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by jonasscheid**
PeakPickerHiRes is one of the [OpenMS](https://openms.readthedocs.io/en/latest/index.html) used for processing proteomics data.
Regarding tests: The tool writes timestamps in the output file, therefore testing md5sums doesn't work.

## PR checklist

Closes #3992  

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by jonasscheid** [comment]
Sorry about the automated requests. I accidentally moved a module folder in another module folder and pushed blindly

File: modules/nf-core/openms/peakpickerhires/meta.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+name: "openms_peakpickerhires"
+description: write your description here
```
Comment (nvnieuwk) [inline modules/nf-core/openms/peakpickerhires/meta.yml:2]: Can you write a tool description?

**Review by nvnieuwk — APPROVED** [review]
One small comment, when fixed you can merge this :partying_face:

File: modules/nf-core/openms/peakpickerhires/meta.yml (hunk 1 above)
Comment (jonasscheid) [reply modules/nf-core/openms/peakpickerhires/meta.yml]: Thanks! missed that one..

**Pull request merged**